internal/xds/rbac/rbac_engine.go

```diff
@@ -65,6 +65,16 @@ func NewChainEngine(policies []*v3rbacpb.RBAC) (*ChainEngine, error) {
 	return &ChainEngine{chainedEngines: engines}, nil
 }
 
+func (cre *ChainEngine) logRequestDetails(rpcData *rpcData) {
+	if logger.V(logLevel) {
```

Comment:
Please remove this `logLevel` const.  It makes it _harder_ to know what verbosity level is being used for these messages.

Reply:
Done.